src/Messages.jl

```diff
@@ -307,8 +307,32 @@ Get header value for `key` (case-insensitive).
 header(m::Message, k, d="") = header(m.headers, k, d)
 header(h::Headers, k::AbstractString, d::AbstractString="") =
     getbyfirst(h, k, k => d, lceq)[2]
-lceq(a,b) = lowercase(a) == lowercase(b)
-
+# lceq(a,b) = lowercase(a) == lowercase(b)
```

Comment:
Delete this comment,
/ make it into a docstring

Might want to use one of our spare unicode `=` symbols for this.
Perhaps `=` with combining subscript L